src/clj/puppetlabs/services/jruby_pool_manager/impl/jruby_internal.clj

```diff
@@ -1,17 +1,24 @@
 (ns puppetlabs.services.jruby-pool-manager.impl.jruby-internal
-  (:require [schema.core :as schema]
-            [puppetlabs.services.jruby-pool-manager.jruby-schemas :as jruby-schemas]
+  (:require [clj-time.core :as time-core]
+            [clj-time.format :as time-format]
+            [clojure.java.io :as io]
+            [clojure.string :as cstring]
```

Comment:
I don't know if I've ever seen this import as anything other than `:as str`. What's the rationale for `cstring`?

Reply:
`str` is already in `clojure.core` so I thought it better to avoid it